pkgs/tools/networking/aws-xray-daemon/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "aws-xray-daemon-${version}";
```

```suggestion
  pname = "aws-xray-daemon";
```

`name` becomes `${pname}-${version}` automatically.